ocs_ci/framework/conf/default_config.yaml

```diff
@@ -68,6 +68,7 @@ DEPLOYMENT:
   # Following options are defaults for deployment from live content
   live_deployment: False
   preserve_bootstrap_node: False
+  noobaa_endpoints: 1
```

Comment:
It would be good to add a docstring or move it a little as docstring above this parameter is:
```
# Following options are defaults for deployment from live content
```
and this is not an option specific for live content deployment.

Reply:
ack, will add